assets/css/utils.sass

```diff
@@ -28,3 +44,40 @@
 .hidden
   display: none
   visibility: hidden
+
```

Comment:
oh i see, this is a generic modal.  well.. i'm gonna move some of the specific stuff like background color, at least, heh